### ISSUE TITLE
Update maintainers guide and use keep-a-changelog

### ISSUE THREAD

**Issue by geerteltink** [zendframework/maintainers#53]
Can we update the maintainers guide and use keep-a-changelog?

I've been using keep-a-changelog a few times for my own projects and it's really easy to use. The maintainers guide could be simplified a lot by incorporating keep-a-changelog.

**Comment by weierophinney** [comment]
Yes, we need to update the guide; we've been using keep-a-changelog everywhere for a couple years now, but I never updated the guide.

When I do, I'll also mention the tool for maintaining the changelog entries.

**Comment by xtreamwayz** [comment]
> When I do, I'll also mention the tool for maintaining the changelog entries.

I didn't mean mentioning the tool, but the actual commands where possible. It's so much easier.
